### PR TITLE
docs: country flag missing in windows

### LINKED ISSUE #16697: docs: Missing country flag in windows
### ESLint version

latest

### What problem do you want to solve?

Similar to https://github.com/eslint/eslint.org/issues/183#issue-1181488254 (happens only in docs page)

### Current
image

### What do you think is the correct solution?

Should show country flags in windows too 

### Expected

image

### Participation

- [X] I am willing to submit a pull request for this change.

### Additional comments

os: windows
browser: chrome

### PR DESCRIPTION
**Pull request by shanpriyan**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)

Fixed missing country flags in the language selector on the docs page

|Before (windows) | After (windows) |
|----|----|
|image|image|

#### Is there anything you'd like reviewers to focus on?

Resolves #16697

### CODE REVIEW

File: docs/src/_includes/layouts/base.html
Comment (amareshsm) [inline docs/src/_includes/layouts/base.html:125]:
```suggestion

        /* Country Flags for Windows */
        @font-face {
            font-family: "Twemoji Country Flags";
            unicode-range: U+1F1E6-1F1FF, U+1F3F4, U+E0062-E0063, U+E0065, U+E0067, U+E006C, U+E006E, U+E0073-E0074, U+E0077, U+E007F;
            src: url("https://cdn.jsdelivr.net/npm/country-flag-emoji-polyfill@0.1/dist/TwemojiCountryFlags.woff2") format("woff2");
        }
```

Pls, lint the files. Run `lint:fix:scss`.

**Review by amareshsm — APPROVED** [review]
LGTM. Thanks! 
Just waiting for the code owner's review.

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

**Pull request merged**
